lib/suggestion-list.coffee

```diff
@@ -76,14 +64,16 @@ class SuggestionList
   onDidConfirm: (fn) ->
     @emitter.on('did-confirm', fn)
 
-  selectNext: =>
+  selectNext: (e) =>
     @emitter.emit('did-select-next')
+    e.stopImmediatePropagation()
```

Comment:
@nathansobo is this ok? It was the only way I could get the editor to not get the event. I feel like there is a better way...